packages/font-glyphs/src/letter/cyrillic/zhe.ptl

```diff
@@ -185,27 +185,27 @@ glyph-block Letter-Cyrillic-Zhe : begin
 
 	foreach { suffix { legShape fSlab fMidSlab } } [Object.entries ZheConfig] : do
 		create-glyph "cyrl/Zhe.\(suffix)" : glyph-proc
-			local df : include : DivFrame para.diversityM 3
+			local df : include : DivFrame para.diversityMM 3
 			include : df.markSet.capital
 			include : Zhe.Shape legShape fSlab fMidSlab df 0 CAP CAP
 
 		create-glyph "cyrl/ZheDescender.\(suffix)" : glyph-proc
-			local df : DivFrame para.diversityM 3
+			local df : DivFrame para.diversityMM 3
 			include [refer-glyph "cyrl/Zhe.\(suffix)"] AS_BASE ALSO_METRICS
 			include : Zhe.Descender fSlab df
 
 		create-glyph "cyrl/zhe.\(suffix)" : glyph-proc
-			local df : include : DivFrame para.diversityM 3
+			local df : include : DivFrame para.diversityMM 3
```

Comment:
Lowercase zhe looks too wide in your samples.

Reply:
Perhaps many more lowercase Cyrillic letters using MM metrics could be reduced back to M metrics, like Nje, Lje, etc.